django/utils/auth.py

```diff
@@ -0,0 +1,10 @@
+from django.conf import settings
+
+
+# When logging out of river, the session is destroyed in the river app
+# but we also need to destroy he session in oprovider, otherwise the client
+# will simply silently re-login. We therefore call the logout route of oprovider.
+# See https://mozilla-django-oidc.readthedocs.io/en/stable/installation.html
+# #log-user-out-of-the-openid-connect-provider
+def logout(request):
+    return settings.OIDC_OP_LOGOUT_ENDPOINT
```

Comment:
Can something bad happen if this is not set?
Also, I'm not sure if there a better place to put that. Don't we have some place where we deal with auth stuff?

Reply:
is `OIDC_OP_LOGOUT_ENDPOINT` the same than `LOGOUT_REDIRECT_URL` ?

Reply:
@tevariou  it's not the same. LOGOUT_REDIRECT_URL is the redirection after the logout has happened. See the code [here](https://github.com/mozilla/mozilla-django-oidc/blob/master/mozilla_django_oidc/views.py#L205)

@Jasopaum yes, something bad happens if this is not set (logout is broken I believe). There will be a warning when the server starts to say that this variable is missing, don't you think it's enough ?
About putting it somewhere else, I think we can regroup it with the auth stuff introduced by this PR https://github.com/arkhn/fhir-river/pull/587 what do you think ?.